test/suites/edgex-no-sec/main_test.go

```diff
@@ -79,9 +92,23 @@ func setup() (teardown func(), err error) {
 		return
 	}
 
+	if err = utils.SnapInstallFromStore(nil, ekuiperSnap, utils.ServiceChannel); err != nil {
+		teardown()
+		return
+	}
+
+	if err = utils.SnapInstallFromStore(nil, ascSnap, utils.ServiceChannel); err != nil {
+		teardown()
+		return
+	}
+
 	// turn security off
 	utils.SnapSet(nil, platformSnap, "security-secret-store", "off")
 	utils.SnapSet(nil, deviceVirtualSnap, "config.edgex-security-secret-store", "false")
+	utils.SnapSet(nil, ascSnap, "app-options", "true")
+	utils.SnapSet(nil, ascSnap, "config.edgex-security-secret-store", "false")
+	utils.SnapSet(nil, ekuiperSnap, "edgex-security", "off")
+	utils.Exec(nil, "sudo rm /var/snap/edgex-ekuiper/current/edgex-ekuiper/secrets-token.json")
```

Comment:
The file deletion should not be needed. If it is, then we have to find a workaround because we can't package a no-sec system and interactively delete files.